stko/utilities/utilities.py

```diff
@@ -1157,3 +1158,98 @@ def vector_angle(vector1, vector2):
     if term <= -1.:
         return np.pi
     return np.arccos(term)
+
+
+def get_torsion_info_angles(
+    mol: stk.ConstructedMolecule,
+    torsion_info
+):
+    """
+    Get the angles for torsion_info in mol.
+
+    The first angle returned is torsion angle in the
+    :class:`stk.ConstructedMolecule`.
+    The second angle returned is torsion angle in the
+    :class:`stk.BuildingBlock`.
+    Both angles are in degrees.
+
+    A :class:`stko.MatchedTorsionCalculator` should yield torsions
+    such that the two angles returned are the same.
+
+    Parameters
+    ----------
+    mol :
+        The constructed molecule for which angles are computed.
+    torsion_info : TorsionInfo
```

Comment:
Add new line.

Reply:
Wasn't sure which line so double check this one.